src/lib/hooks/useDelegateInfo.ts

```diff
@@ -29,11 +33,13 @@ export const useDelegateInfo = (
 
     useEffect(() => {
         (async () => {
-            const delegateName = await getDelegateName(
-                transaction.votes()[0] || transaction.unvotes()[0] || '',
-            );
+            if (votingTypes.includes(type as TransactionType)) {
+                const delegateName = await getDelegateName(
+                    transaction.votes()[0] || transaction.unvotes()[0] || '',
+                );
```

Comment:
Perhaps it would be a good idea to only call this function if an address is obtained

smoething like 

```

const address =  transaction.votes()[0] || transaction.unvotes()[0];`

const delegateName = address ? await getDelegateName(address) : {undefined, empty or null depending of context};
}